R/zchunk_LA107.an_IMAGE_R_C_Sys_Fd_Y.R

```diff
@@ -36,61 +38,251 @@ module_aglu_LA107.an_IMAGE_R_C_Sys_Fd_Y_DISABLED <- function(command, ...) {
+      # drop the country:
+      select(-iso) %>%
+      #group unique identifiers:
+      group_by(GCAM_region_ID, GCAM_commodity, year, system, feed) %>%
+      #sum over the actual values for each unique identifier:
+      summarise(value=sum(value)) ->
+      # store in a table specifying feed consumption by region, commodity, system, feed type, and year:
+      L107.an_Feed_Mt_R_C_Sys_Fd_Y
+
+
+    # Lines 73-80 in original file
+    # Calculate the weighted average feed input-output coefficients by region, commodity, system, feed, and year
+
+    # take the region, commodity, system, feed type, year feed consumption:
+    L107.an_Feed_Mt_R_C_Sys_Fd_Y %>%
+      # add in the corresponding animal production amount as value.y
+      left_join_error_no_match(L107.an_Prod_Mt_R_C_Sys_Fd_Y, by = c("GCAM_region_ID", "GCAM_commodity", "year", "system", "feed")) %>%
+      # calculate the region, commodity, system, feed type, year IO coefficient as feed consumption/animal production
+      mutate(value = value.x/value.y) %>%
+      # drop the columns we don't care about:
+      select(-value.x, -value.y) %>%
+      # replace NA/NaN with the value 100
+      mutate(value=if_else(is.na(value), 100, value))->
+      # store in a table specifying IO coefficients by region, commodity, system, feed type, and year:
+      L107.an_FeedIO_R_C_Sys_Fd_Y
+
+
+
+
+
+
+
+
 
     # Produce outputs
     # Temporary code below sends back empty data frames marked "don't test"
     # Note that all precursor names (in `add_precursor`) must be in this chunk's inputs
     # There's also a `same_precursors_as(x)` you can use
     # If no precursors (very rare) don't call `add_precursor` at all
-    tibble() %>%
-      add_title("descriptive title of data") %>%
-      add_units("units") %>%
-      add_comments("comments describing how data generated") %>%
-      add_comments("can be multiple lines") %>%
+    L107.an_Prod_Mt_R_C_Sys_Fd_Y %>%
+      add_title("Animal production by GCAM region / commodity / system / feed type / year") %>%
+      add_units("Million tonnes (Mt)") %>%
+      add_comments("IMAGE country-level data regarding feed type fraction and mixed versus") %>%
+      add_comments("pastoral system fraction are used to downscale IMAGE country-level total") %>%
+      add_comments("animal production data.") %>%
       add_legacy_name("L107.an_Prod_Mt_R_C_Sys_Fd_Y") %>%
-      add_precursors("precursor1", "precursor2", "etc") %>%
+      add_precursors("common/iso_GCAM_regID",
+                     "temp-data-inject/L100.IMAGE_an_Prodmixfrac_ctry_C_Y",
+                     "temp-data-inject/L100.IMAGE_an_Feedfrac_ctry_C_Sys_Fd_Y",
+                     "temp-data-inject/L100.IMAGE_an_FeedIO_ctry_C_Sys_Y",
+                     "temp-data-inject/L105.an_Prod_Mt_ctry_C_Y") %>%
       # typical flags, but there are others--see `constants.R`
       add_flags(FLAG_NO_TEST, FLAG_LONG_YEAR_FORM, FLAG_NO_XYEAR) ->
```

Comment:
Uh oh--it looks like this FLAG_NO_TEST was left in, which means the chunk's outputs weren't being tested. Can you double-check please?

Note to self: this is probably a stupid thing to have the skeleton script drop in.